exercises/isogram/example.cpp

```diff
@@ -0,0 +1,30 @@
+#include <ctype.h>
+#include <set>
+#include <utility>
+#include "isogram.h"
+
+using std::string;
+
+namespace isogram {
+ 
+bool is_isogram(string const input_str)
+{
+    std::set<char> chars;
```

Comment:
Nit: You don't care about the order, so an unordered_set would be better.
```suggestion
    std::unordered_set<char> chars;
```

Reply:
OK, this adds some clarity.